week_1/main.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>week_1 Assignment</title>
+    <link rel="stylesheet" href="main.css">
+</head>
+<body>
+    <h1 style="padding-left:40px;">
+        My Guest Book
+    </h1>
+    <p>
+        <div class="first line" style="padding-left:40px;">
+            Please sign my guest book. Thanks!
+        </div>
+    </p>
+    <h3 style="padding-left:40px;">
+        <div id="details">
+
+            <b style="padding-right:135px;">What's your name?</b>
+            <input type="text" width="200"><br>
+        </div>
+        
+        <div id="details">    
+            <b style="padding-right:70px">What's your email address?</b>
+            <input type="email" width="200">
+        </div>
+        
+        <div id="details">
```

Comment:
IDs are supposed to be unique. Only one HTML element should have that ID in the entire markup, otherwise it will give errors. Instead use `class="details"` and change the CSS to `.details {...}`

Reply:
Sorry wasn't aware of that , will try to implement in future code. Thank you.